doc/packet_trimming/packet-trimming-design.md

```diff
@@ -301,39 +347,43 @@ The new action will be handled as part of existing `PACKET_ACTION`.
 
 In order to use trimming configuration in ACL rule, a dedicated ACL table type must be defined.
 
-### 2.3.5 Flexcounter orch
+### 2.5.6 Flexcounter orch
 
 Existing infrastructure will be reused.
 
 Flex counter groups `port/queue` will be extended with a new SAI attributes  
 `SAI_PORT_STAT_TRIM_PACKETS/SAI_QUEUE_STAT_TRIM_PACKETS` respectively.
 
-## 2.4 DB schema
+## 2.6 DB schema
 
-### 2.4.1 Config DB
+### 2.6.1 Config DB
 
-#### 2.4.1.1 Switch OA
+#### 2.6.1.1 Switch OA
 ```abnf
 ; defines schema for switch trimming configuration attributes
 key = SWITCH_TRIMMING|GLOBAL ; switch trimming global. Must be unique
 
-; field           = value
-size              = 1*10DIGIT   ; size (in bytes) to trim eligible packet
-dscp_value        = 1*2DIGIT    ; DSCP value assigned to a packet after trimming
-queue_index       = queue-index ; queue index to use for transmission of a packet after trimming
+; field     = value
+size        = 1*10DIGIT   ; size (in bytes) to trim eligible packet
+dscp_value  = dscp-value  ; DSCP value assigned to a packet after trimming
+tc_value    = 1*3DIGIT    ; TC value assigned to a packet after trimming
+queue_index = queue-index ; queue index to use for transmission of a packet after trimming
 
 ; value annotations
+dscp-value = 1*2DIGIT / "from-tc"
 queue-index = 1*3DIGIT / "dynamic"
 ```
 
 **Note:**
 * when packet length is less than `size`, the original full packet will be transmitted out
 * both IPv4/IPv6 packets are updated with `dscp_value` after DSCP remapping
+* when `dscp_value` is set to `from-tc`, the `tc_value` is used for mapping to DSCP
 * when `queue_index` is set to `dynamic`, the `dscp_value` is used for mapping to queue
+* supports runtime configuration change `symmetric->asymmetric` and vice versa
```

Comment:
symmetric-->asymmetric can be done dynamically and the functionality works provided TC_TO_DSCP map is already attached to the egress_port. Please add a note.

Reply:
@kperumalbfn fixed